untrusted/test_enclave/src/lib.rs

```diff
@@ -4,5 +4,41 @@
 /// The test enclave as bytes.
 pub static ENCLAVE: &'static [u8] = include_bytes!(concat!(env!("OUT_DIR"), "/libenclave.signed.so"));
 
-use mc_sgx_urts_sys_types::{sgx_enclave_id_t, sgx_status_t};
+use std::ptr;
+use std::mem::MaybeUninit;
+
+use mc_sgx_urts_sys_types::{sgx_enclave_id_t, sgx_status_t, sgx_report_t, sgx_target_info_t};
```

Comment:
Nit: Sort deps. I thought we had a lint check for this repo?

```suggestion
use mc_sgx_urts_sys_types::{sgx_enclave_id_t, sgx_report_t, sgx_status_t, sgx_target_info_t};
use std::mem::MaybeUninit;
use std::ptr;

```